### PR TITLE
augur-node changed property need to fix in UI.

### PR DESCRIPTION
**Pull request by bthaile**
[Clubhouse Story](https://app.clubhouse.io/augur/story/5906)

description on ch task

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [ ] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

**Comment by tinybike** [comment]
I'm not entirely clear what this PR is for.  The description on CH is just:
```
What are the display conditions for Position Close button.
If position has already been closed?
If 0 shares on position?
Need input from @serena
```

**Comment by bthaile** [comment]
I noticed when Sprinkle did a trading demo and closed an open order. the order 'pending cancellation' didn't get cleared. I found that augur-node changed a property UI needed in order to clear the order. 

The comment about 0 shares is unrelated to the code changes. Not sure why open orders would show 0 shares.

File: src/modules/transactions/actions/construct-transaction.js [hunk 1]
```diff
@@ -139,7 +139,10 @@ export function constructTransferTransaction(log, address) {
 }
 
 export const constructCancelOrderTransaction = (trade, marketId, marketType, description, outcomeId, outcomeName, minPrice, maxPrice, status) => (dispatch, getState) => {
-  const displayPrice = augur.trading.denormalizePrice({ normalizedPrice: trade.price, minPrice, maxPrice })
+  const { orderBooks } = getState()
+  const type = trade.orderType === '0' ? TYPES.BUY : TYPES.SELL
+  const price = !trade.price ? orderBooks[marketId][outcomeId][type][trade.orderId].price : trade.price
```
Comment (tinybike) [inline src/modules/transactions/actions/construct-transaction.js]: I'm not sure I understand what's going on here, is `price` not always included with the trade?  If not, suggest just including it with augur-node's payload.

Note that this code is all significantly modified on the rollback branch, so it's preferable not to also make changes to it here.

File: src/modules/app/actions/listen-to-updates.js [hunk 2]
```diff
@@ -100,12 +100,12 @@ export function listenToUpdates(history) {
           } else {
             console.log('OrderCanceled:', log)
             // if this is the user's order, then add it to the transaction display
-            if (log.orderCreator === getState().loginAccount.address) {
+            if (log.sender === getState().loginAccount.address) {
```
Comment (tinybike) [inline src/modules/app/actions/listen-to-updates.js]: Is `orderCreator` no longer included with the log?

Comment (bthaile) [reply src/modules/app/actions/listen-to-updates.js]: changed to sender from what I see with the newest version of augur-node.

File: src/modules/transactions/actions/construct-transaction.js (hunk 1 above)
Comment (bthaile) [reply src/modules/transactions/actions/construct-transaction.js]: ok, then only changing listen-to-update with property change and re-ordering method calls will fix the issue.

**Comment by bthaile** [comment]
corrections made, only change is to listen-to-updates

**Comment by tinybike** [comment]
LGTM

**Pull request merged**